tests/view/rooms/room/app.js

```diff
@@ -114,4 +116,18 @@ describe('tests/view/rooms/room/app.js, /rooms/{key} =>', () => {
     const redirectUrl = app.getRedirectUrl(href, roomId);
     expect(redirectUrl).to.equal(`/rooms/${roomId}${paramString}`);
   });
+
+  it('ok, "not found modal" element is in html of room from pug file', () => {
+    document.body.innerHTML = ui;
+    expect(document.getElementById('room_not_found_modal').innerHTML).to.not.equal(null);
+  });
+
+  it('ok, "not found modal" is hidden by default', () => {
+    expect(document.getElementById('room_not_found_modal').getAttribute('style')).to.equal('display: none;');
+  });
+
+  it('ok, displayNotFoundModal() shows room not found modal', () => {
+    app.displayNotFoundModal(document);
```

Comment:
here would be nice to check same as my comment above but checking the doc (valid onlicks, style, and not found message